src/tools/rbd/action/Bench.cc

```diff
@@ -293,18 +332,18 @@ int execute(const po::variables_map &vm) {
     return r;
   }
 
-  r = do_bench_write(image, bench_io_size, bench_io_threads, bench_bytes,
-                     bench_random);
+  r = do_bench(image, bench_io_type, bench_io_size, bench_io_threads,
+		     bench_bytes, bench_random);
   if (r < 0) {
-    std::cerr << "bench-write failed: " << cpp_strerror(r) << std::endl;
+    std::cerr << "do_bench failed: " << cpp_strerror(r) << std::endl;
     return r;
   }
   return 0;
 }
 
 Shell::Action action(
-  {"bench-write"}, {}, "Simple write benchmark.", "", &get_arguments, &execute);
+  {"bench"}, {}, "Simple benchmark.", "", &get_arguments, &execute);
```

Comment:
You could leave 'bench-write' and add a 'bench-read' -- each pointing to its own ```execute_bench_write``` or ```execute_bench_read``` method that just invokes ```execute``` with the read or write option.  That way, there are no backwards compatibility issues with users of 'bench-write'